app/models/facility.rb

```diff
@@ -1,6 +1,4 @@
 class Facility < ActiveRecord::Base
-    has_many :facility_items
-    has_many :facility_procedures
-    has_many :items, through: :facility_items
-    has_many :procedures, through: :facility_procedures
+    has_many :procedures
+    has_many :items
```

Comment:
[Style] (https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation)
All indentation in Ruby should be two spaces, not four. You should be able to change the default indentation in your IDE.